policies/azure/AzureCAPI.yaml

```diff
@@ -34,3 +34,20 @@ spec:
           labels:
             +(cluster.x-k8s.io/watch-filter): "capi"
             cluster.x-k8s.io/watch-filter: "capi"
+  - name: kubeadmcontrolplane-enable-allocate-node-cidrs
+    match:
+      resources:
+        kinds:
+          - controlplane.cluster.x-k8s.io/v1alpha3/KubeadmControlPlane
+          - controlplane.cluster.x-k8s.io/v1alpha4/KubeadmControlPlane
+        selector:
+          matchLabels:
+            cluster.x-k8s.io/watch-filter: capi
+    mutate:
+      patchStrategicMerge:
+        spec:
+          kubeadmConfigSpec:
+            clusterConfiguration:
+              controllerManager:
+                extraArgs:
+                  allocate-node-cidrs: "true"
```

Comment:
Just to double check, is the intent to only default or also override an existing value?

Reply:
I want to override. The only value that works is "true", anything else will generate a broken cluster

Reply:
Cool, then this looks perfect